snapcraft/ua_manager.py

```diff
@@ -63,6 +73,17 @@ def _attach(ua_token: str) -> None:
         raise UATokenAttachError from error
 
 
+def _enable_services(services: List[str]) -> None:
+    """Enable the specified UA services."""
+    try:
+        with emit.open_stream("Enable UA services") as stream:
+            subprocess.check_call(
+                ["ua", "enable", *services, "--beta"], stdout=stream, stderr=stream
```

Comment:
why are we hardcoding --beta here?

Reply:
If the entire concept is beta, I would argue that this should require an experimental flag

Reply:
It allows us to enable any service regardless of the beta status. Otherwise I can verify each service individually to see if beta is required using a hardcoded table lookup, and then add beta if necessary -- it's easier to just add it in all cases.

Reply:
Just to follow up from chat, we agreed to use --enable-experimental-ua-services for now

Reply:
Some services require `--beta`, others don't. ROS services, specifically, require it (the primary use of this feature will be to enable ROS services). Added the `--enable-experimental-ua-services` flag.